src/os/Transaction.cc

```diff
@@ -493,11 +493,13 @@ void Transaction::dump(ceph::Formatter *f)
         ghobject_t oid = i.get_oid(op->oid);
         uint64_t expected_object_size = op->expected_object_size;
         uint64_t expected_write_size = op->expected_write_size;
+        uint32_t alloc_hint_flags = op->alloc_hint_flags;
         f->dump_string("op_name", "op_setallochint");
         f->dump_stream("collection") << cid;
         f->dump_stream("oid") << oid;
         f->dump_stream("expected_object_size") << expected_object_size;
         f->dump_stream("expected_write_size") << expected_write_size;
+        f->dump_stream("alloc_hint_flags") << alloc_hint_flags;
```

Comment:
why do you prefer using `dump_stream()` over `dump_unsigned()`? how about dump the flags using `dump_format()` in hex? as this field is or'ed flags of `CEPH_OSD_ALLOC_HINT_FLAG_*`

Reply:
Thanks, inspired by your comment I changed the dump format, acceptable?